frontend/admin/src/js/components/dashboard/Dashboard.tsx

```diff
@@ -66,15 +90,28 @@ const OpenMenuButton: React.FC<OpenMenuButtonProps> = ({
 
 interface DashboardProps {
   contentRoutes: Routes<RouterResult>;
+  loggedIn: boolean;
```

Comment:
All these props are just for the router.  I wonder if the `useRouter` call with its stuff should get lifted to the parent and just pass in `content`.

Reply:
Just saw this, I tend to agree. I'd like to see if someone with more familiarity could weigh in. To me this makes sense so we can avoid excessive prop drilling.

Reply:
Actually, why not just let useRouter access the AuthorizationContext and AuthenticationContext itself?

Reply:
Smart!  Added in bd1fb90a461ed6e17f57cffb07d6a931b097dbd2 and I like it much better.